fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -304,6 +295,7 @@ public void close() {
     @VisibleForTesting
     <R> CompletableFuture<R> runAsync(@Nonnull final Function<FDBRecordStore, CompletableFuture<R>> function,
```

Comment:
It would probably be worth changing the comment above this into a javadoc, and describing the parameters

Reply:
Done.